ShovIP/app/src/androidTest/java/com/example/shovip/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.shovip
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
+    @Test
+    fun useAppContext() {
+        // Context of the app under test.
+        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
+        assertEquals("com.example.shovip", appContext.packageName)
+    }
+}
```

Comment:
I think this example file can be removed

Reply:
@shovian please take care of this

Reply:
- Removed
-

Reply:
@shovian it is still here for me, did you remove it on another branch?

Reply:
yes